### PR TITLE
3125 - Enable print function on the Pan-European platform as on the FRA Platform

### LINKED ISSUE #3125: panEuropean -> Enable print function on the pan-European platform as on the FRA platform

### PR DESCRIPTION
**Pull request by yaguzmang**
- Removed the FraPrint component to use the generic Print component instead.
- Moved the existing print translations to `<asessment name>.print.<key>`.
- Added Pan European print translations (to be revised) to `en/panEuropean/print.js` and imported them into `en/panEuropean/panEuropean.js`
- Renamed a couple of "fra-print" css variables to a more generic "assessment-print".

https://github.com/openforis/fra-platform/assets/41337901/1151d593-c3e2-4d0c-b19c-6c7291ff364d

[http://fra-data.local:9000/assessments/panEuropean/2020/AUT/print](http://fra-data.local:9000/assessments/panEuropean/2020/AUT/print):

fra-data local_9000_assessments_panEuropean_2020_AUT_print (1)

### CODE REVIEW

File: src/client/pages/Print/Print.tsx [hunk 1]
```diff
@@ -1,25 +1,77 @@
...
+import { useGetDescriptionValues } from './hooks/useGetDescriptionValues'
+import { useGetTableSections } from './hooks/useGetTableSections'
+import TableOfContent from './TableOfContent'
 
 const Print: React.FC = () => {
-  const assessment = useAssessment()
+  const { assessmentName } = useCountryRouteParams()
+
+  const { t } = useTranslation()
+  const countryIso = useCountryIso()
```
Comment (minotogna) [inline src/client/pages/Print/Print.tsx]: countryIso should be now taken from `useCountryRouteParams<CountryIso>()`

File: src/client/pages/Print/Print.tsx [hunk 2]
```diff
@@ -1,25 +1,77 @@
...
 
-import { useAssessment } from 'client/store/assessment'
+import { useCountry } from 'client/store/area'
+import { useCycle } from 'client/store/assessment'
+import { useSections } from 'client/store/metadata'
+import { useCountryIso } from 'client/hooks'
+import { useIsPrintRoute } from 'client/hooks/useIsRoute'
+import { useCountryRouteParams } from 'client/hooks/useRouteParams'
 import Loading from 'client/components/Loading'
+import { useGetTableData } from 'client/pages/Print/hooks/useGetTableData'
```
Comment (minotogna) [inline src/client/pages/Print/Print.tsx]: inner hooks, use convention to import from relative path

File: src/client/pages/Print/Print.tsx [hunk 3]
```diff
@@ -1,25 +1,77 @@
...
+import Section from 'client/pages/Section'
 
-import FraPrint from './FraPrint'
-
-const Components: Record<string, React.FC> = {
-  [AssessmentNames.fra]: FraPrint,
-}
+import { useGetDescriptionValues } from './hooks/useGetDescriptionValues'
+import { useGetTableSections } from './hooks/useGetTableSections'
+import TableOfContent from './TableOfContent'
```
Comment (minotogna) [inline src/client/pages/Print/Print.tsx]: instead, components absolute path

File: src/client/pages/Print/Print.tsx [hunk 4]
```diff
@@ -1,25 +1,77 @@
...
+
+  let title = ''
+  if (onlyTables) title = t(`${assessmentName}.print.titleTables`, { cycleName: cycle.name })
+  if (!onlyTables && deskStudy) title = `${t(`assessment.${assessmentName}`)} ${t('assessment.deskStudy')}`
+  if (!onlyTables && !deskStudy) title = t(`${assessmentName}.print.title`, { cycleName: cycle.name })
+
+  return (
+    <div className="assessment-print__container">
+      <div>
+        <div className="assessment-print__header">
```
Comment (minotogna) [inline src/client/pages/Print/Print.tsx]: css name can be simplified by removing assessment. they can just be print__container and print__header

File: src/i18n/resources/en/panEuropean/print.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  title: 'Pan-European {{cycleName}} report',
+  titleTables: 'Pan-European {{cycleName}} reporting tables',
```
Comment (minotogna) [inline src/i18n/resources/en/panEuropean/print.js]: note. panEuropean , not Pan-European

**Comment by minotogna** [comment]
good job 👍 just some minor comments

File: src/client/pages/Print/Print.tsx (hunk 2 above)
Comment (yaguzmang) [reply src/client/pages/Print/Print.tsx]: Missed this. Thanks!

File: src/client/pages/Print/Print.tsx (hunk 3 above)
Comment (yaguzmang) [reply src/client/pages/Print/Print.tsx]: Done, thanks!

File: src/i18n/resources/en/panEuropean/print.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+  disclaimerGenerated: `This document was generated automatically using the report made available as a contribution to the FAO Global Forest Resources Assessment {{cycleName}}, and submitted to FAO as an official government document. The content and the views expressed in this report are the responsibility of the entity submitting the report to FAO. FAO cannot be held responsible for any use made of the information contained in this document.`,
+  disclaimerGeneratedDeskStudy: `This document was generated automatically from a desk study. FAO cannot be held responsible for any use made of the information contained in this document.`,
```
Comment (minotogna) [inline src/i18n/resources/en/panEuropean/print.js:7]: @yaguzmang thinking more about this: can we show disclaimers only if assessmentName === AssessmentNames.fra ?

**Review by minotogna — APPROVED** [review]

**Pull request merged**